playbooks/templates/DigitalShadows_SearchLight_fetch_alerts.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Fetch alerts from Digital Shadows SearchLight",
+    "uuid": "367ead9d-9e0f-42e7-abc2-1c134bb95a3e",
```

Comment:
I forgot that one on my first review
```suggestion
```

Reply:
Review applied in commit 475c02a01f26e5584f937eba5ce3ec6710350d0d